include/astshim/Mapping.h

```diff
@@ -90,7 +90,7 @@ class Mapping : public Object {
     Is this an inverted mapping?
 
     Note: this gets the @ref Mapping_Invert "Invert" attribute.
-    This method is not called `getInvert` because that is too similar to @ref getInverse.
+    This method is not called `getInvert` because that is too similar to @ref inverted.
```

Comment:
I'm not sure this comment makes sense anymore: `isInverted` is closer to `inverted` than `getInvert` would be. I don't think it's worth changing the API (ie, renaming `isInverted` to `getInvert`), although if you were keen to do that I wouldn't object, but I do think we should remove the comment.

Reply:
I think it important to say why it is not called `getInvert` because most of astshim has `getX` for every attribute `X`. However, I will change it to say "because that sounds like it might return the inverse" which is my main objection to `getInvert`.